integration-tests/test_obfuscation.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import subprocess
+import os
+import yaml
+from time import sleep
+import re
+
+pytestmark = pytest.mark.usefixtures("register_subman")
+CONTENT_REDACTION_FILE = "/etc/insights-client/file-content-redaction.yaml"
+
+
+def test_obfuscation(insights_client, tmp_path):
```

Comment:
this test always fails, and the open bugs for it shows this too; hence, please use the `pytest.mark.xfail()` marker to mark this test as "expected to fail"

Reply:
Bug https://issues.redhat.com/browse/RHINENG-8846 is going to merged, so l think we can leave this code unchanged. Because bug will be already fixed when we use this test case in regular testing.

Reply:
That fix will not be available for 6/7 days yet, and I would like to merge this PR (which does not have left concerns from me) rather than waiting for some days... :) In the end, removing the xfail marker can be done quickly/easily later on.

WDYT?

Reply:
Sure, pls merge this PR.